bindinfo/session_handle_test.go

```diff
@@ -16,6 +16,7 @@ package bindinfo_test
 
 import (
 	"context"
+	"github.com/pingcap/tidb/testkit/testdata"
```

Comment:
group import

Reply:
sorry

Reply:
the error shows "try make bazel_prepare"